util_test.go

```diff
@@ -391,6 +391,12 @@ func skipOnJenkins(t *testing.T, message string) {
 	}
 }
 
+func runOnlyOnDockerContainer(t *testing.T, message string) {
+	if os.Getenv("AUTHENTICATION_TESTS_ENV") == "" {
```

Comment:
Again - I don't see this variable to be set in `docker run` command, how does it work?